test/sandbox/test/view_component_system_test.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class ViewComponentSystemTest < ViewComponent::SystemTestCase
```

Comment:
@joelhawksley let me know what you think about this API change based on our conversation. Once this is looking good, I'll update the corresponding README.